storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -502,7 +518,7 @@ func (c *PrometheusConverter) addSummaryDataPoints(ctx context.Context, dataPoin
 
 		pt := dataPoints.At(x)
 		timestamp := convertTimeStamp(pt.Timestamp())
-		baseLabels, err := createAttributes(resource, pt.Attributes(), scope, settings, nil, false, metadata)
+		baseLabels, err := createAttributes(resource, pt.Attributes(), scope, settings, nil, false, metadata, 0)
```

Comment:
[Nit] Could we use the corresponding constant for improved readability, and not rely on the constant's value?
```suggestion
		baseLabels, err := createAttributes(resource, pt.Attributes(), scope, settings, nil, false, metadata, pmetric.AggregationTemporalityUnspecified)
```